src/avs/AVSDetailsHeader.jsx

```diff
@@ -0,0 +1,152 @@
+};
+
+export default function AVSDetailsHeader({ avs }) {
+  const [state, dispatch] = useMutativeReducer(reduceState, {
+    isCopied: false,
+    copyTimeout: null
+  });
+
+  const handleCopy = useCallback(value => {
+    if ('clipboard' in navigator) {
+      navigator.clipboard.writeText(value).catch();
+
+      if (state.copyTimeout) {
+        clearTimeout(state.copyTimeout);
+      }
+
+      dispatch({
+        isCopied: true,
+        copyTimeout: setTimeout(() => dispatch({ isCopied: false }), 2000)
+      });
+    }
+  }, []);
+
+  return (
+    <div className="bg-content1 border border-outline rounded-lg w-full p-4 break-words">
+      <div className="flex items-center">
+        <AVSLogo avs={avs} />
+        <span className="ml-2 text-foreground-1 text-3xl font-display font-medium">
+          {avs.metadata?.name ?? 'N/A'}
+        </span>
+        {/*TODO: implement ranking when coming from list view & accessing directly avs*/}
+        <div className="text-xs ml-2 p-1 rounded-md bg-foreground-2 text-content1">
+          # 1
+        </div>
+      </div>
+      <div className="my-4 text-xs text-foreground-1 break-words">
+        {avs.metadata?.description ?? 'N/A'}
+      </div>
+
+      {avs.metadata?.website && (
+        <Link
+          href={avs.metadata.website}
+          target="_blank"
+          rel="noreferrer"
+          className="text-xs text-secondary"
+        >
+          {avs.metadata.website}
+        </Link>
+      )}
+
+      <div className="flex">
+        <div className="flex items-center">
+          <Link
+            href={formatEtherscanLink(avs.address)}
+            target="blank"
+            rel="noreferrer"
+            className="text-xs text-secondary"
+          >
+            {avs.address.substr(0, 6)}...
+            {avs.address.substr(-4)}
+          </Link>
+          <Button
+            variant="ghost"
+            isIconOnly
+            className="border-none ml-1"
+            size="sm"
+            onClick={() => handleCopy(avs.address)}
+          >
+            {state.isCopied ? (
+              <span
+                className="material-symbols-outlined text-secondary text-2xl"
+                style={{
+                  fontVariationSettings: `'FILL' 0, 'wght' 400, 'GRAD' 0, 'opsz' 20`
+                }}
+              >
+                check
+              </span>
+            ) : (
+              <span className="material-symbols-outlined text-secondary text-2xl">
+                content_copy
+              </span>
+            )}
+          </Button>
+        </div>
+
+        {avs.metadata?.twitter !== '' && (
+          <Button
+            variant="ghost"
+            as={Link}
+            href={avs.metadata.twitter}
+            target="_blank"
+            rel="noreferrer"
+            size="sm"
+            isIconOnly
+            className="ml-12 border-none text-secondary"
+          >
+            <TwitterIcon />
+          </Button>
+        )}
```

Comment:
Slightly shorter way with a Twitter handle and smaller icon:

```suggestion
        {!!avs.metadata?.twitter && (
          <Button
            variant="ghost"
            as={Link}
            href={avs.metadata.twitter}
            target="_blank"
            rel="noreferrer"
            size="sm"
            className="ml-12 border-none text-secondary"
          >
            <span
              className="bg-secondary h-4 w-4"
              style={{ mask: 'url(/assets/x.svg) no-repeat' }}
            ></span>
            @
            {avs.metadata.twitter.substring(
              avs.metadata.twitter.lastIndexOf('/') + 1
            )}
          </Button>
        )}
```

Reply:
Updated it with backgroud-color, otherwise the icon is not visible on the dark background